#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *
from math import pi

PACKAGE_NAME = "squirrel_navigation"

gen = ParameterGenerator()
gen.add("plan_with_footprint", bool_t, 0, "", False)
gen.add("plan_with_constant_heading", bool_t, 0, "", False)
gen.add("heading", double_t, 0, "", 0.0, -pi, pi)
gen.add("collision_based_replanning", bool_t, 0, "", False)
gen.add("collision_based_replanning_lookahead", double_t, 0, "", 1.0, 0.0, 1000.0)
gen.add("verbose", bool_t, 0, "", False)

exit(gen.generate(PACKAGE_NAME, "squirrel_navigation", "GlobalPlanner"))